financial-templates-lib/clients/ExpiringMultiPartyClient.js

```diff
@@ -33,6 +33,14 @@ class ExpiringMultiPartyClient {
   // result this function will return positions that are undercollateralized due to too little collateral or a withdrawal
   // that, if passed, would make the position undercollateralized.
   getUnderCollateralizedPositions = tokenRedemptionValue => {
+    if (!tokenRedemptionValue) {
```

Comment:
Same comment as below.

Reply:
This one I can remove because we check for `price !== null` in the calling client before making a call to `getUnderCollateralizedPositions`